### DISCUSSION TITLE
主题 Next 里面的三个按钮如何配置使用呢？

### DISCUSSION THREAD

**Discussion by volmodaoist** [tangly1024/NotionNext#2847] [Q&A]
<img width="703" alt="image" src="https://github.com/user-attachments/assets/f35b0997-5f17-4ec2-8335-bbad1cc4e30e">

图中的三个按钮「精选必看」「热门文章」「实用教程」如何实用呢？

**Comment by LooseLi** [comment]
你好，需要先在 Notion 中添加以下三个标签：必看精选、热门文章、实用教程。然后，为想要展示的文章设置相应的标签即可。如图：
<img width="909" alt="image" src="https://github.com/user-attachments/assets/12973392-09e4-4095-bfa8-09da921d37fb" />
